src/combobox/combobox.component.ts

```diff
@@ -151,6 +158,14 @@ export class ComboBox implements OnChanges, AfterViewInit, AfterContentInit {
 	 * Sets the optional helper text.
 	 */
 	@Input() helperText: string | TemplateRef<any>;
+	/**
+	 * Set to `true` for invalid state.
+	 */
+	@Input() invalid = false;
+	/**
+	 * Value displayed if dropdown is in invalid state.
+	 */
+	@Input() invalidText = "";
```

Comment:
We usually eventually need string or template options.
Any reason why that won't be the case here?